runtime/runtime/src/lib.rs

```diff
@@ -1330,6 +1330,17 @@ impl Runtime {
             vec![]
         };
 
+        // Remove the only testnet account with large storage key.
+        if ProtocolFeature::RemoveAccountWithLongStorageKey.protocol_version() == protocol_version
+            && migration_flags.is_first_block_with_chunk_of_version
+        {
+            let account_id = "contractregistry.testnet".parse().unwrap();
```

Comment:
Do we know who owns this account? Is there any money in there?

Reply:
10 testnet tokens, `eve.testnet` is creator https://testnet.nearblocks.io/address/contractregistry.testnet

Reply:
I'm wondering if any invariants e.g. about total near supply may break if we just remove it. If that's the case perhaps we could simulate this as some special transaction. That's me being super cautious I don't know if there is any real issue.

Reply:
Not really... Total supply is increased by 5% every year, relation between burned gas, validator rewards and thus total balance is very implicit.